src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -419,7 +419,7 @@ def subvolume_getpath(self, fs_handle, **kwargs):
                 if not path:
                     raise VolumeException(
                         -errno.ENOENT, "Subvolume '{0}' not found".format(subvolname))
-                ret = 0, path, ""
+                ret = 0, path.decode("utf-8"), ""
```

Comment:
see also 3d63cd947f55a8508f972ee249ea6a04836429ae

Reply:
yeh -- that's the commit that introduced this change. For this fix, we are only interested in calls that return to ceph-mgr DaemonServer.